docs/config/configuration.md

```diff
@@ -202,6 +203,9 @@ bitcoin-s {
         # before we timeout
         addressQueueTimeout = 5 seconds
         
+        # How often the wallet will rebroadcast unconfirmed transactions
+        rebroadcastFrequency = 30 minutes
```

Comment:
This seems like an aggressive default setting. Maybe like 4 hour re-tries? It ends up being 6 times a day? Frankly if it isn't getting confirmed, it's probably too low of a fee, which requires user interaction to bump. Re-trying the broadcast isn't going to get it confirmed.

Reply:
Changed to 4 hours